styleguide/guides/issues.rst

```diff
@@ -0,0 +1,172 @@
+Asking more specific questions often helps in this situation.
+
+The following question(s), helped us in the past, to get a better handle on this:
+
+* What is the cost/benefit ratio of creating an issue?
+    * cost being mainly time and effort
+    * benefit being the upsides we gain from having an issue
+
+* What is the signal to noise ratio?
+    * signal being the actual code/change bringing some benefit/value to the project
+    * noises being the overhead
+
+* Is the user (api or product) impacted by this change?
+    * Is it an "internal improvement" quality of life and/or code quality
+
+
+Examples
+--------
+Fixing a couple of typos
+________________________
+You stumble upon a couple of typos in a Project.
+
+Possible Action(s)
+~~~~~~~~~~~~~~~~~~
+* You fix the typos without creating an issue
+
+Suspicious amount of typos
+___________________________
+You stumble upon a various typos within a piece of documentation you are reading. Because of the vast amount
+of typos and the overall structure you expect the whole documentation to contain a substantial amount of typos.
+
+Possible Action(s)
+~~~~~~~~~~~~~~~~~~
+* If the documentation is not to big and you can make the time to fix the typos right away without creating an issue.
+* If the documentation is quite big, and fixing this e.g. would take you more than 1/2 a day you definitely should create an issue.
+* You are unsure how long it will take to address and check the documentation and you currently can't spent time for detours create an issue.
+
+What makes a good issue
++++++++++++++++++++++++
+
+    **TL;DR:** An good issue describes a piece of work, it's context and it's assumption in a way, any person capable of working
+    on the project, can understand it, based on the information referenced and within the issue.
+
+To begin with, we want to make sure you understand that creating good a issues is not a zero effort action. Various things
+like issue templates, can remove tedious boilerplate work, but still creating a good issue is some actual work.
+It is crucial to have well crafted issues, because bad issues do way more harm than good. So let's take a step
+back and let's think about why we need to have good issues, in order to better understand what actually makes a good issue.
+
+Boiled down to it's core issue tracking is just a list of pieces of work which are needed to be done.
+This list in turn, than can be used to plan, organize and track "progress/work".
+
+Broken down to an individual issue, this means an issue should provide all relevant information about
+a "single" piece/unit of work.
+
+Even though the part what an issue is about is well understood, by most people, this is also where it goes wrong.
+
+**Why?** Because we as humans tend to use and assume context implicitly. Assuming context implicitly already can be a problem
```

Comment:
That's why I start my issues with the background. Should we emphasize this more in the templates?